ArduPlane/systemid.cpp

```diff
@@ -204,42 +215,100 @@ void AP_SystemID::update()
             attitude_offset_deg.z = waveform_sample;
             break;
         case AxisType::RECOVER_ROLL:
```

Comment:
I wonder if we should add new axis for this. It would just make the tools easier as we would not have to check flight mode too.

Reply:
@IamPete1 Not sure what you mean?  for the recover_XXX cases?  please explain further

Reply:
Just if we should have a `FIXEDWING_RECOVER_ROLL` to differentiate from the existing `RECOVER_ROLL` that is used for the VTOL controllers. We log the axis type in the `SIDD` log although I'm not sure if that log is still needed in this case.

Reply:
@IamPete1 Starting to see the wisdom in your comment now that I am trying to code up the web tool. This would make that much easier